core/appeal/service.go

```diff
@@ -525,6 +531,97 @@ func (s *Service) Revoke(ctx context.Context, id string, actor, reason string) (
 	return revokedAppeal, nil
 }
 
+func (s *Service) BulkRevoke(ctx context.Context, filters *domain.ListAppealsFilter, actor, reason string) ([]*domain.Appeal, error) {
+	result := make([]*domain.Appeal, 0)
+	appeals, err := s.Find(ctx, filters)
```

Comment:
we can hardcode the list appeals filter.status to be only `active` since we should only revoke active appeals, and also to reduce load to db

Reply:
done